bedrock/firefox/views.py

```diff
@@ -616,8 +616,10 @@ def get_template_names(self):
                 template = 'firefox/developer/whatsnew.html'
             else:
                 template = 'firefox/whatsnew/index.html'
-        elif version.startswith('83.') and ftl_file_is_active('firefox/whatsnew/whatsnew-fx83'):
-            template = 'firefox/whatsnew/whatsnew-fx83.html'
+        elif version.startswith('83.') and locale.startswith('de'):
```

Comment:
I think instead of `startswith` this should be an equal, since we only have one German: `locale == 'de'`